consensus/istanbul/backend/backend_test.go

```diff
@@ -63,7 +56,8 @@ func TestBackend_GetTargetReceivers(t *testing.T) {
 	configItems = append(configItems, epoch(3))
 	configItems = append(configItems, governanceMode("single"))
 	configItems = append(configItems, minimumStake(new(big.Int).SetUint64(4000000)))
-	configItems = append(configItems, istanbulCompatibleBlock(new(big.Int).SetUint64(5)))
+	configItems = append(configItems, lowerBoundBaseFee(2))
+	configItems = append(configItems, upperBoundBaseFee(10))
```

Comment:
Why are these values ​​used?

Reply:
Oh it's no use.. removed those values: https://github.com/kaiachain/kaia/pull/557/commits/f441f1dba0f21b0ff951f976177b89c9ad947fb2